docs/next/guides/internationalization/layout-direction.md

```diff
@@ -0,0 +1,220 @@
+| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------
+| Starting and ending edges                                                                         | The left-hand edge is considered the starting edge of the grid, and the right-hand edge is considered the ending edge of the grid.                               | The left-hand edge is considered the starting edge of the grid, and the right-hand edge is considered the ending edge of the grid.
+| The order of columns on the screen                                                                | The cell rendering flows from the left-hand side of the screen to the right-hand side. Thus, the cell at coordinates (0, 0) is rendered in the top-left corner.  | The cell rendering flows from the right-hand side of the screen to the left-hand side. Thus, the cell at coordinates (0, 0) is rendered in the top-right corner.
+| The text direction in the cells                                                                   | All cells inherit the LTR direction from the container element.                                                                                                  | All cells inherit the RTL direction from the container element.
+| The position of the row headers                                                                   | The row headers are rendered on the left-hand edge of the grid.                                                                                                  | The row headers are rendered on the right-hand edge of the grid.
+| The position of the [frozen columns](@/guides/columns/column-freezing.md)                         | The columns are frozen at the right-hand edge of the grid.                                                                                                       | The columns are frozen at the right-hand edge of the grid.
+| [Keyboard navigation](@/guides/accessories-and-menus/keyboard-navigation.md)                      | <kbd>Tab</kbd> moves to the right, and <kbd>Shift</kbd> + <kbd>Tab</kbd> moves to the left.                                                                      | <kbd>Tab</kbd> moves to the left, and <kbd>Shift</kbd> + <kbd>Tab</kbd> moves to the right.
+| The position of the [selection](@/guides/cell-features/selection.md) handles                      | The circular selection handles on mobile devices appear in the top-left and bottom-right corners of the selection border.                                        | The circular selection handles on mobile devices appear in the top-right and bottom-left corners of the selection border.
```

Comment:
I propose to complete the table and add info about `fillHandle` corner position.
```suggestion
| The position of the [selection](@/guides/cell-features/selection.md) handles                      | The circular selection handles on mobile devices appear in the top-left and bottom-right corners of the selection border.                                        | The circular selection handles on mobile devices appear in the top-right and bottom-left corners of the selection border.
| The position of the selection [fill handler](@/api/options.md#fillHandle) corner                      | The fill handle rectangle corner appears on the bottom-right corner of the selection.                                        | The fill handle rectangle corner appears on the bottom-left corner of the selection
```